README.md

```diff
@@ -1,24 +1,24 @@
-Artemis
+Wynntils
 ========
 [![Discord](https://discordapp.com/api/guilds/394189072635133952/widget.png)](https://discord.gg/ve49m9J)
 [![CurseForge](https://cf.way2muchnoise.eu/short_wynntils.svg)](https://www.curseforge.com/minecraft/mc-mods/wynntils)
 [![Modrinth](https://img.shields.io/modrinth/dt/Wynntils?label=modrinth)](https://modrinth.com/mod/wynntils)
 [![GitHub](https://img.shields.io/github/downloads/Wynntils/Artemis/total?logo=github)](https://github.com/Wynntils/Artemis/releases)
-[![License](https://img.shields.io/badge/license-LGPL%203.0-green.svg)](https://github.com/Wynntils/Artemis/blob/main/LICENSE)
+[![License](https://img.shields.io/badge/license-LGPL%203.0-green.svg)](https://github.com/Wynntils/Wynntils/blob/main/LICENSE)
 
 <div align="center">
-<img src="https://upload.wikimedia.org/wikipedia/commons/d/d2/Artemis.png" width=20%>
 
-*(Image in public domain)*
-</div>
+<img src="http://cdn.wynntils.com/wynntils%20logo%20-%20w%20paintstroke.png" width=15% alt="Wynntils Logo">
+
```

Comment:
We need to move the `</div>` to right after the img, otherwise part of the text gets centered as well. 

(Or maybe that was your intention? It looks odd, imho, though)

Reply:
It was kind of the intention. Sure, Im okay with it either way.

Reply:
Thanks. We can go over the README separately afterwards, and see if we can highlight this paragraph in another way that does not look too odd.